modules/renter/download.go

```diff
@@ -520,3 +524,53 @@ func (r *Renter) DownloadHistory() []modules.DownloadInfo {
 	}
 	return downloads
 }
+
+// ClearDownloadHistory clears the renter's download history based on
+// provided start and end times
+func (r *Renter) ClearDownloadHistory(start, end time.Time) error {
+	if err := r.tg.Add(); err != nil {
+		return err
+	}
+	defer r.tg.Done()
+	r.downloadHistoryMu.Lock()
+	defer r.downloadHistoryMu.Unlock()
+
+	// Clear download history if both start and end times are zero values
+	if start.Equal(time.Unix(0, 0)) && end.Equal(time.Unix(0, 0)) {
```

Comment:
you can use `start.IsZero()` for this

Reply:
:+1:

Reply:
turns out `time.Unix(0, 0).IsZero()` is false, so I'll leave it as `time.Unix(0, 0)`